Index: compat-wireless-2014-01-23.1/net/mac80211/scan.c
===================================================================
--- compat-wireless-2014-01-23.1.orig/net/mac80211/scan.c	2014-01-30 16:46:25.000000000 +0100
+++ compat-wireless-2014-01-23.1/net/mac80211/scan.c	2014-02-14 11:02:53.348029618 +0100
@@ -27,7 +27,7 @@
 
 #define IEEE80211_PROBE_DELAY (HZ / 33)
 #define IEEE80211_CHANNEL_TIME (HZ / 33)
-#define IEEE80211_PASSIVE_CHANNEL_TIME (HZ / 9)
+#define IEEE80211_PASSIVE_CHANNEL_TIME (HZ / 33)
 
 void ieee80211_rx_bss_put(struct ieee80211_local *local,
 			  struct ieee80211_bss *bss)
